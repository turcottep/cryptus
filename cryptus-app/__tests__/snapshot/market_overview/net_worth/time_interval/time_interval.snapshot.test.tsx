import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import TimeInterval from "../../../../../components/market/net_worth/time_interval/time_interval";

describe("<TimeInterval />", () => {
  it("should display properly", async () => {
    const test_active = 0;
    const test_cb = async () => {};
    React.useState = jest.fn().mockReturnValueOnce([test_active, {}]);

    const tree = renderer
      .create(<TimeInterval active={test_active} callback={test_cb} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
