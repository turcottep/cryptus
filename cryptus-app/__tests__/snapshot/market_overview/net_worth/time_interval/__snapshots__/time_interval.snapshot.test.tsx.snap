// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimeInterval /> should display properly 1`] = `
<div>
  <div
    className="undefined undefined"
    onClick={[Function]}
  >
    1W
  </div>
  <div
    className="undefined "
    onClick={[Function]}
  >
    1M
  </div>
  <div
    className="undefined "
    onClick={[Function]}
  >
    3M
  </div>
  <div
    className="undefined "
    onClick={[Function]}
  >
    1Y
  </div>
  <div
    className="undefined "
    onClick={[Function]}
  >
    ALL
  </div>
</div>
`;
